store/src/components/kiosk/menu.js

```diff
@@ -382,7 +401,8 @@ const KioskMenu = props => {
                                        />
                                     </div>
                                     <span tw="whitespace-nowrap text-[var(--hern-primary-color)] text-2xl font-bold">
-                                       Subtotal -{' '}
+                                       {config?.menuSettings?.cartTitle?.value}{' '}
+                                       -{' '}
```

Comment:
Please add a default value 
```suggestion
                                       {config?.menuSettings?.cartTitle?.value || 'Subtotal'}{' '}
                                       -{' '}
```